src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -106,7 +106,6 @@ public function current_node() {
 	 *
 	 * @see https://html.spec.whatwg.org/#has-an-element-in-the-specific-scope
 	 *
-	 * @param string   $tag_name         Name of tag check.
```

Comment:
Oh, I missed this; this looks accidental, no?

Reply:
yeah, thanks. Not sure what happened here.
added back in 64008211b0